lib/matplotlib/_mathtext.py

```diff
@@ -2159,10 +2162,17 @@ def operatorname(self, s, loc, toks):
         next_char = next((c for c in s[next_char_loc:] if c != ' '), '')
         delimiters = self._left_delim | self._ambi_delim | self._right_delim
         delimiters |= {'^', '_'}
+
         if (next_char not in delimiters and
```

Comment:
Actually, it looks like avoiding insertion of the spurious space is just adding a check that `if (next_char != '' and ...)` (i.e. we are not at the end of the math string).  Perhaps you can make that change as part of this PR too?  This way we would avoid changing the baseline images.

Reply:
I don't think this actually fixes the issue. I made the change, reverted to the original images locally, and the same failures occur. I believe changing this statement could alter spacing after non sub/super scripted operators. However, not for sub/superscripted ones, since they will be followed by a '_' or '^' by definition. 

In essence, we would need to check the next char after the sub/superscripted content, but it's very hard to tell where the sub/super scripted content stops.

Reply:
@anntzer any thoughts on how you want to move forward?

Reply:
Sorry, indeed the above solution is not enough.  Perhaps we can just drop that test for now (replace the entry by None, and delete the baseline images): indeed, there's already tests 37, 56, and 75 which test placement of integrals limits, which seems enough.